public/wp-content/themes/newsup-child/single-anime.php

```diff
@@ -66,18 +70,148 @@
             </div>
 		      <?php } ?>
 
-           <div class="media mg-info-author-block">
-            <?php $newsup_enable_single_post_admin_details = esc_attr(get_theme_mod('newsup_enable_single_post_admin_details','true'));
-            if($newsup_enable_single_post_admin_details == true) { ?>
-            <a class="mg-author-pic" href="<?php echo esc_url(get_author_posts_url( get_the_author_meta( 'ID' ) ));?>"><?php echo get_avatar( get_the_author_meta( 'ID') , 150); ?></a>
-                <div class="media-body">
-                  <h4 class="media-heading"><?php esc_html_e('By','newsup'); ?> <a href "<?php echo esc_url(get_author_posts_url( get_the_author_meta( 'ID' ) ));?>"><?php the_author(); ?></a></h4>
-                  <p><?php the_author_meta( 'description' ); ?></p>
-                </div>
-              <?php } ?>
-            </div><?php $newsup_enable_related_post = esc_attr(get_theme_mod('newsup_enable_related_post','true'));
+<!--                    list genres and anime information here-->
+                    <?php
+                    $genres = get_field('anime_genres'); // relationship (other post type)
```

Comment:
Add a condition here to *not* load a relationship if there are no genres defined. I don't have any defined on my db and I'm getting an error.

![error](https://user-images.githubusercontent.com/48684825/78667671-5ccc8400-78a7-11ea-8821-ddc2477ece3e.png)

Reply:
anime_genre is a required field, but sure, add this.

Reply:
Yeah i think it's still worth it to put since I have the ability to make an anime without the required field.
We could probably make a new issue for that.

Reply:
that error not making any sense to me

![image](https://user-images.githubusercontent.com/38733594/79075986-6c433700-7cc4-11ea-90e2-3fdb50f3e91a.png)

i have checks for all the query objects and all the grabbed fields, 

i think your error is something else, maybe a db migration issue.
do you want me to use a try catch when i call get_field() ?

i also dont know how to make an anime post without the required fields in order to test the issue XD

Reply:
`Call to undefined function get_field()` is saying that you do not have ACF installed. ACF plugin defines `get_field()`.

Reply:
> `Call to undefined function get_field()` is saying that you do not have ACF installed. ACF plugin defines `get_field()`.

yea, i was thinking this too

Reply:
Yep yall right. I had it installed, but I didn't have the relationship set up. All good.